docs/configuring-a-resource.md

```diff
@@ -557,13 +559,66 @@ p.AddResourceConfigurator("aws_autoscaling_group", func(r *config.Resource) {
 })
 ```
 
+### Initializers
+
+Initializers establish ownership of the supplied Managed resources in Crossplane.
+This typically involves the operations that are run before calling any
+ExternalClient methods.
+
+This configuration option will provide that setting custom initializers for per 
+resource and is set by using the [InitializerFns] field that is a list of [NewInitializerFn]:
+
+```go
+// NewInitializerFn returns the Initializer with a client.
+type NewInitializerFn func(client client.Client) managed.Initializer
+```
+
+Initializer is an interface in [crossplane-runtime]:
+
+```go
+type Initializer interface {
+	Initialize(ctx context.Context, mg resource.Managed) error
+}
+```
+
+So, an interface must be passed to the related configuration field for adding initializers for a resource.
+Let's check an example about how can this configuration be used.
+
+`tags` is very common schema field for aws resources. Also, setting [some external labels] is a very common
+use case in Crossplane. Therefore, an initializer that sets the crossplane related labels for resources
+can be added in config file of the related resource:
+
+```go
+// Tagger implements the Initialize function to set external tags
+type Tagger struct {
+    kube      client.Client
+}
+
+// TagInitializer returns a tagger to use default tag initializer.
+var TagInitializer NewInitializerFn = func(kube client.Client) managed.Initializer {
+    return &Tagger{kube: kube}
+}
+
+func (t *Tagger) Initialize(ctx context.Context, mg xpresource.Managed) error {
+	// ...
+}
+p.AddResourceConfigurator("aws_s3_bucket", func(r *config.Resource) {
+    r.InitializerFns = append(r.InitializerFns, TagInitializer)
+}
+```
+
+In above example, `Tagger` struct implements the `Initialize` function, and so a `Tagger` instance
+can be used as an initializer. The custom logic of the initializer will be implemented in `Initialize`
+function.
+
 [comment]: <> (References)
 
 [Terrajet]: https://github.com/crossplane/terrajet
 [External name]: #external-name
 [Cross Resource Referencing]: #cross-resource-referencing
 [Additional Sensitive Fields and Custom Connection Details]: #additional-sensitive-fields-and-custom-connection-details
-[Late Initialization Behavior]: #late-initialization-behavior
```

Comment:
This reference is wrong, so the link does not work in document. Fixed with this PR.